### PR TITLE
DXE-2232 Add M1 support for gtm

### PR DESCRIPTION
**Pull request by mimazaka**
Add support of cli-gtm on Apple Silicon based machines.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # CLI-GTM Release Notes
```
Comment (Slonimskaia) [inline CHANGELOG.md:2]: Please add release date here as next Wednesday 10.05

File: README.md [hunk 2]
```diff
@@ -22,14 +22,12 @@ for your system, or by cloning this repository and compiling it yourself.
 
 If you want to compile it from source, you will need Go 1.14 or later.
 
-1. Fetch the package:  
-  `go get github.com/akamai/cli-gtm`
-2. Change to the package directory:  
-  `cd $GOPATH/src/github.com/akamai/cli-gtm`
-3. Compile the binary:
+1. Create a clone of the target repository:  
+  `go clone https://github.com/akamai/cli-gtm.git`
```
Comment (Slonimskaia) [inline README.md:26]:
```suggestion
  `git clone https://github.com/akamai/cli-gtm.git`
```

File: README.md [hunk 3]
```diff
@@ -22,14 +22,12 @@ for your system, or by cloning this repository and compiling it yourself.
...
+2. Compile the binary:
    - Linux/macOS/nix: `go build -o akamai-gtm`
    - Windows: `go build -o akamai-gtm.exe`
-4. Move the binary (`akamai-gtm` or `akamai-gtm.exe`) in to your `PATH`
+3. Move the binary (`akamai-gtm` or `akamai-gtm.exe`) in to your `PATH`
```
Comment (Slonimskaia) [inline README.md:30]:
```suggestion
```

Comment (Slonimskaia) [reply README.md:30]: Third step is not mandatory

**Review by Slonimskaia — CHANGES_REQUESTED** [review]
Do not merge till release date which is on 10th of May

File: akamai-gtm.go [hunk 4]
```diff
@@ -19,7 +19,7 @@ import (
 )
 
 var (
-	VERSION = "0.4.0"
+	VERSION = "0.5.0"
```
Comment (wzagrajcz) [inline akamai-gtm.go:22]: Shouldn't we change the version (0.4.0 -> 0.5.0) during release? Or are we merging the M1 PR with release candidate already?

Comment (Slonimskaia) [reply akamai-gtm.go:22]: we will merge it only on release date https://github.com/akamai/cli-gtm/pull/12#pullrequestreview-1413307940
We need to have all the changes here to create binaries for release

File: README.md [hunk 5]
```diff
@@ -22,14 +22,11 @@ for your system, or by cloning this repository and compiling it yourself.
...
 If you want to compile it from source, you will need Go 1.14 or later.
 
-1. Fetch the package:  
-  `go get github.com/akamai/cli-gtm`
-2. Change to the package directory:  
-  `cd $GOPATH/src/github.com/akamai/cli-gtm`
-3. Compile the binary:
+1. Create a clone of the target repository:  
+  `git clone https://github.com/akamai/cli-gtm.git`
+2. Compile the binary:
```
Comment (wzagrajcz) [inline README.md:27]:
```suggestion
2. Change to the package directory and compile the binary:
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,10 @@
...
 
+## Version 0.5.0(May 10,2023)
```
Comment (Slonimskaia) [inline CHANGELOG.md:3]:
```suggestion
## Version 0.5.0 (May 10,2023)
```

File: CHANGELOG.md (hunk 1 above)
Comment (Slonimskaia) [reply CHANGELOG.md:2]: It is not valid anymore since we decided to have release branch

File: README.md [hunk 7]
```diff
@@ -22,14 +22,11 @@ for your system, or by cloning this repository and compiling it yourself.
 
 If you want to compile it from source, you will need Go 1.14 or later.
```
Comment (Slonimskaia) [inline README.md:23]:
```suggestion
If you want to compile it from source, you will need Go 1.18 or later.
```

**Review by Slonimskaia — APPROVED** [review]

**Pull request merged**